source/includes/configuration/_02-get-historical-snapshot.md.erb

```diff
@@ -1,12 +1,18 @@
 ## Get configuration
 
 ```shell
-$ curl 'https://ci.example.com/go/api/admin/config/5059cf548db9ea2d1b9192b45529ccf0.xml' \
+$ curl 'https://ci.example.com/go/api/admin/config.xml' \
       -u 'username:password'
 ```
 
 > The above command returns the contents of the config file
 
+
```

Comment:
Other api's show this information. Kept it there to indicate the extra header and the content type.
For e.g at
https://api.go.cd/current/#create-pipeline